### PR TITLE
Add CWL description, bring other logging info up to date

### PR DESCRIPTION
**Pull request by LeoKuhorev**
- Add Continuous Web (Debug) logs description
- Bring other logging-related information up to date

### CODE REVIEW

File: Teams/log-files.md [hunk 1]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
-This article describes these logs and how they are used. For information about troubleshooting specific issues, see: [Teams Troubleshooting](/MicrosoftTeams/troubleshoot/teams). 
+This article describes these logs and how they are used. For information about troubleshooting specific issues, see: [Teams Troubleshooting](/MicrosoftTeams/troubleshoot/teams).
 
 For information on collecting logs from Microsoft Teams Rooms devices, see [Download device logs](/microsoftteams/rooms/rooms-manage#download-device-logs).
 
 For information about how to contact support, see [Get support](/microsoft-365/business-video/get-help-support).
 
 > [!NOTE]
-> In this article, the term **Debug logs** refers to the logs that are used for troubleshooting. However, the files that are generated for these logs will contain the term **diagnostic logs** in their names.  
+> In this article, the term **Debug logs** (aka Web ) refers to the logs that are used for troubleshooting. However, the files that are generated for these logs will contain the term **diagnostic logs** in their names.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:37]:
```suggestion
> In this article, the term **Debug logs** (also called Web logs) refers to the logs that are used for troubleshooting. However, the files that are generated for these logs will contain the term **diagnostic logs** in their names.
```

File: Teams/log-files.md [hunk 2]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
 
 When creating a support request with Microsoft Support, the support engineer will require the debug logs. Having the debug logs on hand before creating the support request will allow Microsoft to quickly start troubleshooting the problem. **Media** or **Desktop** logs are only required if requested by Microsoft.
 
-Debug, Desktop, and Media logs will be collected in one folder with the name _MSTeams Diagnostics Log \<local date and time\>_. This folder can be compressed and shared when you open a support request with Microsoft Support. The folder will contain folders for Desktop, Meeting (Media), and Debug (web). You can collect the files using the following keyboard shortcuts:
+### Log collection
 
-- Windows:
-  <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>1</kbd>
+Debug/Web, Desktop, and Media logs will be collected in one folder with the name _MSTeams Diagnostics Log \<local date and time\>_ which will be placed in your Downloads directory. This folder can be compressed and shared when you open a support request with Microsoft Support. The folder will contain folders for Desktop, Meeting (Media), and Debug (web).
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:47]:
```suggestion
Debug/Web, Desktop, and Media logs will be collected in one folder with the name _MSTeams Diagnostics Log \<local date and time\>_ which will be placed in your Downloads directory. This folder can be compressed and shared when you open a support request with Microsoft Support. The folder will contain folders for Desktop, Meeting (Media), and Debug (Web).
```

File: Teams/log-files.md [hunk 3]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
 
-- Mac:
-  <kbd>Option</kbd> + <kbd>Command</kbd> + <kbd>Shift</kbd> + <kbd>1</kbd>
+To collect logs for Windows:
+ - Click the Microsoft Teams icon in your system tray and select **Collect support files**.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:50]:
```suggestion
 - Select the Microsoft Teams icon in your system tray and then select **Collect support files**.
```

File: Teams/log-files.md [hunk 4]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
 
+To collect logs for Mac:
+ - Click the Help menu in Microsoft Teams, and select **Collect support files**.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:53]:
```suggestion
 - Select the Help menu in Microsoft Teams and then select **Collect support files**.
```

File: Teams/log-files.md [hunk 5]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
+
+To collect logs for Linux:
+ - Click on the Microsoft Teams icon in your system tray, and select **Get Logs**.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:56]:
```suggestion
 - Select the Microsoft Teams icon in your system tray and then select **Get Logs**.
```

File: Teams/log-files.md [hunk 6]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
-The following table outlines the various clients and their associated logs. Log files are stored in locations specific to the client and operating system.
-
-|Client |Debug|Desktop|Media|
-|---------|---------|---------|---------|
-|Web    |X         |-         |-         |
-|Windows     |X         |X         |X         |
-|Mac OSX     |X         |X         |X         |
-|iOS     |-         |-         |-         |
-|Android     |-         |-         |-         |
+The following table outlines the various clients and their associated logs. Prior to being collected, log files are stored in the teams app folder with the folder's location being specific to the client and operating system: Windows: `%appdata%\Microsoft\Teams\`
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:64]:
```suggestion
The following table outlines the various clients and their associated logs. Prior to being collected, log files are stored in the Teams app folder with the folder's location being specific to the client and operating system.
- Windows: `%appdata%\Microsoft\Teams\`
```

File: Teams/log-files.md [hunk 7]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
+Mac: `~/Library/Application Support/Microsoft/Teams/`
+Linux: `~/.config/Microsoft/Microsoft Teams/`
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:66]:
```suggestion
- Mac: `~/Library/Application Support/Microsoft/Teams/`
- Linux: `~/.config/Microsoft/Microsoft Teams/`
```

File: Teams/log-files.md [hunk 8]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
+|iOS      |-        |-                |-        |-        |
+|Android  |-        |-                |-        |-        |
+
 
 For a complete list of supported operating systems and browsers, see [Get clients for Microsoft Teams](get-clients.md).
 
 ## Debug logs
 
-Debug logs are produced by the Windows and Mac desktop clients, as well as by browser-based clients. The logs are text-based and are read from the bottom-up. They can be read using any text-based editor, and new logs are created when logging into the client.
+Debug logs (also called Web logs) are produced by the Windows and Mac desktop clients, as well as by browser-based clients. The logs are text-based and are read from the bottom-up. They can be read using any text-based editor, and new logs are created when logging into the client.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:81]:
```suggestion
Debug logs (also called Web logs) are produced by the Windows and Mac desktop clients, as well as by browser-based clients. The logs are text-based and are read from the bottom up. They can be read using any text-based editor, and new logs are created when logging into the client.
```

File: Teams/log-files.md [hunk 9]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
 
-To collect logs for Linux:
-- Keyboard shortcut: <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>1</kbd>  
-- The files will be available in `~/Downloads`
+## Continuous Debug logs
 
-To collect logs for Browser and Windows:
-- Keyboard shortcut: <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Shift</kbd> + <kbd>1</kbd>  
-- The files will be available in `%userprofile%\Downloads`
+Continuous Debug (aka Web) logging is a feature that helps capture log data by automatically transferring logs from memory to the file system in a continuous fashion. On average there will be a full day's worth of log data available when enabled. The logs are read from the top-down.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:90]:
```suggestion
Continuous Debug logging (also called Web logging) is a feature that helps capture log data by automatically transferring logs from memory to the file system in a continuous fashion. On average there will be a full day's worth of log data available when enabled. The logs are read from the top down.
```

File: Teams/log-files.md [hunk 10]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
+
+Continuous Debug/Web logging is turned on by default on teams desktop for computers if your CPU is:
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:92]:
```suggestion
Continuous Debug/Web logging is turned on by default on Teams desktop for computers if your CPU is:
```

File: Teams/log-files.md [hunk 11]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
+ - Intel Core i3 and Core i5 (generation 10 or higher), Core i7 and Core i9 (any generation) excluding the following suffixes: U, Y, G1-G7, UE, UL, M, QM
+- Intel Xeon
+- AMD Ryzen 5, 7, 9 and TR series excluding U suffix
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:95]:
```suggestion
 - Intel Core i3 and Core i5 (generation 10 or higher), Core i7 and Core i9 (any generation), excluding the following suffixes: U, Y, G1-G7, UE, UL, M, and QM
- Intel Xeon
- AMD Ryzen 5, 7, 9, and TR series, excluding the U suffix
```

File: Teams/log-files.md [hunk 12]
```diff
@@ -23,75 +23,85 @@ appliesto:
...
+- Apple Silicon
+
+Otherwise, it is turned off by default.
+
+### Admin Configuration
+Managing Continuous Debug/Web logging for your end users provides a seamless troubleshooting experience, especially when issues are intermittent. Admins can use the TeamsMediaLoggingPolicy cmdlet to enable and manage Continuous Debug/Web logging for users.
+
+Read [Grant-CsTeamsMediaLoggingPolicy](/powershell/module/teams/grant-csteamsmedialoggingpolicy) for PowerShell cmdlets and more information.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:103]:
```suggestion
See [Grant-CsTeamsMediaLoggingPolicy](/powershell/module/teams/grant-csteamsmedialoggingpolicy) for PowerShell cmdlets and more information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @LeoKuhorev, kindly commit the suggested changes. Many thanks.

**Comment by LeoKuhorev** [comment]
> Copy review - @LeoKuhorev, kindly commit the suggested changes. Many thanks.

Thanks a lot for your review @JohanFreelancer9 . Applied suggested changes, please take another look when you get a chance

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**